site/templates/nav.jet

```diff
@@ -2,73 +2,67 @@
-                </li>
-                {{end}}
-              {{else}}
-                <li class="nav-item {{len(.Link.ExternalURL) == 0 && routeToSlug(.Link.Slug) == currentUrlPath ? "active" : ""}}">
-                  <a class="nav-link" href="{{len(.Link.ExternalURL) > 0 ? .Link.ExternalURL : routeToSlug(.Link.Slug)}}">{{.Label}}</a>
-                </li>
-              {{end}}
-            {{end}}
-          </ul>
-        {{end}}
-
-        <div class="navbar-nav s72-hide">
-          <s72-language-selector></s72-language-selector>
-        	{{ yield nav_user_logged_out() }}
-        	{{ yield nav_user_logged_in() }}
-        </div>
-
+      <s72-language-selector></s72-language-selector>
+      <div class="navbar-nav navbar-logged-out s72-hide">
+        {{ yield nav_user_logged_out() }}
+      </div>
+      <div class="navbar-nav navbar-logged-in s72-hide">
+        {{ yield nav_user_logged_in() }}
       </div>
-  </nav>
 
-  {{if len(site.Navigation.Header) > 0 && showSubNav}}
-    <ul class="sub-nav">
-      {{range site.Navigation.Header}}
-        {{if len(.Items) > 0}}
-          <li class="nav-item">
-            <s72-dropdown>
-              <a class="s72-dropdown-toggle nav-link" href="{{len(.Link.ExternalURL) > 0 ? .Link.ExternalURL : len(.Items) == 0 ? routeToSlug(.Link.Slug) : "#"}}">{{.Label}}</a>
-              <div class="s72-dropdown-menu sub-nav-menu">
-                {{range sub := .Items}}
-                  <a class="nav-link" href="{{len(sub.Link.ExternalURL) > 0 ? sub.Link.ExternalURL : routeToSlug(sub.Link.Slug) }}">{{sub.Label}}</a>
-                {{end}}
-              </div>
-            </s72-dropdown>
-          </li>
-        {{else}}
-          <li class="nav-item {{len(.Link.ExternalURL) == 0 && routeToSlug(.Link.Slug) == currentUrlPath ? "active" : ""}}">
-            <a class="nav-link" href="{{len(.Link.ExternalURL) > 0 ? .Link.ExternalURL : routeToSlug(.Link.Slug)}}">{{.Label}}</a>
-          </li>
-        {{end}}
+      {{if len(site.Navigation.Header) > 0 && showSubNav}}
+        <ul class="sub-nav">
+          {{range site.Navigation.Header}}
+            {{if len(.Items) > 0}}
+              <li class="nav-item">
+                <s72-dropdown>
+                  <a class="s72-dropdown-toggle nav-link" href="{{len(.Link.ExternalURL) > 0 ? .Link.ExternalURL : len(.Items) == 0 ? routeToSlug(.Link.Slug) : "#"}}">{{.Label}}</a>
+                  <div class="s72-dropdown-menu sub-nav-menu">
+                    {{range sub := .Items}}
+                      <a class="nav-link" href="{{len(sub.Link.ExternalURL) > 0 ? sub.Link.ExternalURL : routeToSlug(sub.Link.Slug) }}">{{sub.Label}}</a>
+                    {{end}}
+                  </div>
+                  <svg  width="24" height="24" viewBox="0 0 24 24" class="dropdown-caret"><path d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z" /></svg>
```

Comment:
We use Font Awesome 4 for icons... no need for svg chaos!

Reply:
Replaced with Font Awesome